libraries/shaders/src/shaders/Shaders.h

```diff
@@ -155,8 +155,8 @@ struct Source {
     static const Source& get(uint32_t shaderId);
 
 private:
-    // Disallow copy construction and assignment
-    Source(const Source& other) = default;
+    // Disallow copy construction
+    Source(const Source& other) = delete;
```

Comment:
Is this being assigned somewhere? What is correct here, the code or the previous comment of "copy construction and assignment"?

Reply:
The assignment operator is being declared higher up, I had initially put in Q_DISBALE_COPY in here but the compiler complained as it deletes both copy constructor and assignment.  The comment is therefore... wrong.

I don't know why there's a =default in here, it's private so I suspect it has about as much effect as if it weren't there.  If it was defined or used I would have gotten a compiler error trying to build this.